src/components/preprint-card.js

```diff
@@ -93,7 +94,7 @@ export default function PreprintCard({
   return (
     <Fragment>
       <div
-        className={classNames('preprint-card', { 'preprint-card--new': isNew })}
+        className={classNames('preprint-card', { 'preprint-card--new': isNew && !isDashboardCard })}
```

Comment:
We need this because the dashboard doesn't use box-shadow for the preprint cards (on hover).